libreyolo/label/assist.py

```
@@ -0,0 +1,242 @@
+            from PIL import Image
+
+            with Image.open(image_path) as im:
+                iw, ih = im.size
+            xywhn = [
+                [((x1 + x2) / 2) / iw, ((y1 + y2) / 2) / ih, (x2 - x1) / iw, (y2 - y1) / ih]
+                for x1, y1, x2, y2 in bn.xyxy
+            ]
+        cls_arr, conf_arr = bn.cls, bn.conf
+        model_names = getattr(r, "names", {}) or {}
+        out: List[dict] = []
+        for (cx, cy, w, h), c, p in zip(xywhn, cls_arr, conf_arr):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)